### PR TITLE
Update backup/restore guide.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@chris1984 if you have time, would you mind comparing mind alterations to this with your experience as well?

File: docs/user_guide/backup/index.md [hunk 1]
```diff
@@ -43,7 +44,7 @@ tar --selinux -czvf config_files.tar.gz \
 /etc/sysconfig/elasticsearch \
 /root/ssl-build \
 /var/www/html/pub \
-/usr/share/katello/candlepin-cert.crt
+/var/lib/puppet/foreman_cache_data
```
Comment (cfouant) [inline docs/user_guide/backup/index.md]: I believe some of the data files are outdated, and others are missing. When creating the backup scripts, these are the files I came up with:
tar --selinux -czvf config_files.tar.gz \
/etc/katello/ \
/etc/katello-installer/ \
/etc/candlepin/ \
/etc/foreman/ \
/etc/foreman-proxy/ \
/etc/gutterball/ \
/etc/hammer/ \
/etc/pulp/ \
/etc/puppet/ \
/etc/qpid/ \
/etc/qpid-dispatch/ \
/etc/elasticsearch/ \
/etc/tomcat6/ \
/etc/pki/katello/ \
/etc/pki/katello-certs-tools/ \
/etc/sysconfig/katello \
/etc/sysconfig/elasticsearch \
/etc/sysconfig/tomcat6 \
/root/ssl-build/ \
/var/lib/puppet/foreman_cache_data \
/var/www/html/pub/

Comment (ehelms) [inline docs/user_guide/backup/index.md]: I don't think we need `/etc/katello` or `/etc/sysconfig/katello`

**Comment by ehelms** [comment]
Closing this in favor of a future PR from @cfouant to update this with the scripts

**Pull request closed without merging**